compilation error "error: inlining failed in call to always_inline" on archlinux

```
In file included from /tmp/makepkg/rpcs3-git/src/rpcs3/rpcs3/stdafx.h:28,
                 from /tmp/makepkg/rpcs3-git/src/build/rpcs3/Emu/cotire/rpcs3_emu_CXX_prefix.cxx:4,
                 from /tmp/makepkg/rpcs3-git/src/build/rpcs3/Emu/cotire/rpcs3_emu_CXX_prefix.hxx:4,
                 from <command-line>:
/tmp/makepkg/rpcs3-git/src/rpcs3/Utilities/types.h: In instantiation of ‘constexpr To std::bit_cast(const From&) [with To = v128; From = v128; <template-parameter-1-3> = void]’:
/tmp/makepkg/rpcs3-git/src/rpcs3/Utilities/BEType.h:454:34:   required from ‘static v128 se_storage<T, 16, 16>::to(const T&) [with T = v128]’
/tmp/makepkg/rpcs3-git/src/rpcs3/Utilities/BEType.h:502:30:   required from ‘se_t<T, true, Align>& se_t<T, true, Align>::operator=(se_t<T, true, Align>::type) [with T = v128; long unsigned int Align = 16; se_t<T, true, Align>::type = v128]’
/tmp/makepkg/rpcs3-git/src/rpcs3/rpcs3/Emu/Cell/PPUInterpreter.cpp:3636:53:   required from here
/tmp/makepkg/rpcs3-git/src/rpcs3/Utilities/BEType.h:7:19: note: ‘union v128’ has no user-provided default constructor
    7 | union alignas(16) v128
      |                   ^~~~
/tmp/makepkg/rpcs3-git/src/rpcs3/Utilities/BEType.h:9:7: note: and the implicitly-defined constructor does not initialize ‘char v128::_bytes [16]’
    9 |  char _bytes[16];
      |       ^~~~~~
In file included from /usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/immintrin.h:37,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/x86intrin.h:32,
                 from /tmp/makepkg/rpcs3-git/src/rpcs3/Utilities/types.h:6,
                 from /tmp/makepkg/rpcs3-git/src/rpcs3/rpcs3/stdafx.h:27,
                 from /tmp/makepkg/rpcs3-git/src/build/rpcs3/Emu/cotire/rpcs3_emu_CXX_prefix.cxx:4,
                 from /tmp/makepkg/rpcs3-git/src/build/rpcs3/Emu/cotire/rpcs3_emu_CXX_prefix.hxx:4,
                 from <command-line>:
/usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/smmintrin.h: In static member function ‘static bool ppu_interpreter::FCTIWZ(ppu_thread&, ppu_opcode_t)’:
/usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/smmintrin.h:447:1: error: inlining failed in call to always_inline ‘int _mm_extract_epi32(__m128i, int)’: target specific option mismatch
  447 | _mm_extract_epi32 (__m128i __X, const int __N)
      | ^~~~~~~~~~~~~~~~~
/tmp/makepkg/rpcs3-git/src/rpcs3/rpcs3/Emu/Cell/PPUInterpreter.cpp:4833:61: note: called from here
 4833 |  ppu.fpr[op.frd] = std::bit_cast<f64, s64>(_mm_extract_epi32(res, 0));
      |                                            ~~~~~~~~~~~~~~~~~^~~~~~~~
In file included from /usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/immintrin.h:37,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/x86intrin.h:32,
                 from /tmp/makepkg/rpcs3-git/src/rpcs3/Utilities/types.h:6,
                 from /tmp/makepkg/rpcs3-git/src/rpcs3/rpcs3/stdafx.h:27,
                 from /tmp/makepkg/rpcs3-git/src/build/rpcs3/Emu/cotire/rpcs3_emu_CXX_prefix.cxx:4,
                 from /tmp/makepkg/rpcs3-git/src/build/rpcs3/Emu/cotire/rpcs3_emu_CXX_prefix.hxx:4,
                 from <command-line>:
/usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/smmintrin.h:447:1: error: inlining failed in call to always_inline ‘int _mm_extract_epi32(__m128i, int)’: target specific option mismatch
  447 | _mm_extract_epi32 (__m128i __X, const int __N)
      | ^~~~~~~~~~~~~~~~~
/tmp/makepkg/rpcs3-git/src/rpcs3/rpcs3/Emu/Cell/PPUInterpreter.cpp:4833:61: note: called from here
 4833 |  ppu.fpr[op.frd] = std::bit_cast<f64, s64>(_mm_extract_epi32(res, 0));
      |                                            ~~~~~~~~~~~~~~~~~^~~~~~~~
In file included from /usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/immintrin.h:37,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/x86intrin.h:32,
                 from /tmp/makepkg/rpcs3-git/src/rpcs3/Utilities/types.h:6,
                 from /tmp/makepkg/rpcs3-git/src/rpcs3/rpcs3/stdafx.h:27,
                 from /tmp/makepkg/rpcs3-git/src/build/rpcs3/Emu/cotire/rpcs3_emu_CXX_prefix.cxx:4,
                 from /tmp/makepkg/rpcs3-git/src/build/rpcs3/Emu/cotire/rpcs3_emu_CXX_prefix.hxx:4,
                 from <command-line>:
/usr/lib/gcc/x86_64-pc-linux-gnu/9.1.0/include/smmintrin.h:447:1: error: inlining failed in call to always_inline ‘int _mm_extract_epi32(__m128i, int)’: target specific option mismatch
  447 | _mm_extract_epi32 (__m128i __X, const int __N)
      | ^~~~~~~~~~~~~~~~~
/tmp/makepkg/rpcs3-git/src/rpcs3/rpcs3/Emu/Cell/PPUInterpreter.cpp:4833:61: note: called from here
 4833 |  ppu.fpr[op.frd] = std::bit_cast<f64, s64>(_mm_extract_epi32(res, 0));
      |                                            ~~~~~~~~~~~~~~~~~^~~~~~~~
make[2]: *** [rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/build.make:743: rpcs3/Emu/CMakeFiles/rpcs3_emu.dir/Cell/PPUInterpreter.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
```

https://aur.archlinux.org/packages/rpcs3-git/
https://github.com/RPCS3/rpcs3/issues/1546

--- a/rpcs3/Emu/Cell/PPUInterpreter.cpp
+++ b/rpcs3/Emu/Cell/PPUInterpreter.cpp
@@ -4830,7 +4830,7 @@ bool ppu_interpreter::FCTIWZ(ppu_thread& ppu, ppu_opcode_t op)
 {
 	const auto b = _mm_load_sd(&ppu.fpr[op.frb]);
 	const auto res = _mm_xor_si128(_mm_cvttpd_epi32(b), _mm_castpd_si128(_mm_cmpge_pd(b, _mm_set1_pd(0x80000000))));
-	ppu.fpr[op.frd] = std::bit_cast<f64, s64>(_mm_extract_epi32(res, 0));
+	ppu.fpr[op.frd] = std::bit_cast<f64, s64>(_mm_cvtsi128_si32(res));
 	if (UNLIKELY(op.rc)) fmt::throw_exception("%s: op.rc", __func__); //ppu_cr_set(ppu, 1, ppu.fpscr.fg, ppu.fpscr.fl, ppu.fpscr.fe, ppu.fpscr.fu);
 	return true;
 }
@@ -5022,7 +5022,7 @@ bool ppu_interpreter::FCTID(ppu_thread& ppu, ppu_opcode_t op)
 {
 	const auto b = _mm_load_sd(&ppu.fpr[op.frb]);
 	const auto res = _mm_xor_si128(_mm_set1_epi64x(_mm_cvtsd_si64(b)), _mm_castpd_si128(_mm_cmpge_pd(b, _mm_set1_pd(f64(1ull << 63)))));
-	ppu.fpr[op.frd] = std::bit_cast<f64>(_mm_extract_epi64(res, 0));
+	ppu.fpr[op.frd] = std::bit_cast<f64>(_mm_cvtsi128_si64(res));
 	if (UNLIKELY(op.rc)) fmt::throw_exception("%s: op.rc", __func__); //ppu_cr_set(ppu, 1, ppu.fpscr.fg, ppu.fpscr.fl, ppu.fpscr.fe, ppu.fpscr.fu);
 	return true;
 }
@@ -5031,7 +5031,7 @@ bool ppu_interpreter::FCTIDZ(ppu_thread& ppu, ppu_opcode_t op)
 {
 	const auto b = _mm_load_sd(&ppu.fpr[op.frb]);
 	const auto res = _mm_xor_si128(_mm_set1_epi64x(_mm_cvttsd_si64(b)), _mm_castpd_si128(_mm_cmpge_pd(b, _mm_set1_pd(f64(1ull << 63)))));
-	ppu.fpr[op.frd] = std::bit_cast<f64>(_mm_extract_epi64(res, 0));
+	ppu.fpr[op.frd] = std::bit_cast<f64>(_mm_cvtsi128_si64(res));
 	if (UNLIKELY(op.rc)) fmt::throw_exception("%s: op.rc", __func__); //ppu_cr_set(ppu, 1, ppu.fpscr.fg, ppu.fpscr.fl, ppu.fpscr.fe, ppu.fpscr.fu);
 	return true;
 }

